debian/copyright

```diff
@@ -0,0 +1,620 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
As FFmpeg is deployed using GPL licence and not LGPL under debian systems, I think avTranscoder will be use in this case with the GPL licence. Same avec TuttleOFX, it requires a multi licencing due to links with multi licencing 3rd party.